src/Mapper/Mapper.test.ts

```diff
@@ -354,17 +354,48 @@ describe("Mapper", () => {
 
       afterEach(() => {
         pushRecordStub.restore();
+        MockModel.prototype.fields.organization.serialize = false;
+        MockModel.prototype.relationships.organization.modelRelatedFieldName = undefined;
       });
 
-      it("normalizes nested related data", async () => {
+      it("normalizes nested related data with information about the parent model", async () => {
         const normalizeStub = stub(fakeRelatedService.mapper, "normalize").callThrough();
         await mapper.normalize(rawModelData);
-        expect(normalizeStub.firstCall.args[0]).to.equal(relatedModelData);
+        expect(normalizeStub.firstCall.args[0]).to.deep.equal({
+          ...relatedModelData,
+          parentServiceName: "fakeModel",
+        });
       });
 
-      it("creates a pushRecord action with related data", async () => {
+      it("creates a pushRecord action with the related data and info about the parent model", async () => {
```

Comment:
Where is the "info about the parent model"?

Reply:
Good catch! That test was redundant so I removed it.